Dockerfile.ci

```diff
@@ -186,7 +199,7 @@ RUN mkdir -pv ${AIRFLOW_HOME} && \
     mkdir -pv ${AIRFLOW_HOME}/logs
 
 # Increase the value here to force reinstalling Apache Airflow pip dependencies
-ARG PIP_DEPENDENCIES_EPOCH_NUMBER="4"
+ARG PIP_DEPENDENCIES_EPOCH_NUMBER="5"
```

Comment:
Don't do that :). It's only needed if you remove dependencies. In this case it's not needed but it's not harmful because you also add the MS SQL drivers few lines above (so the cache is cleared anyway).

Reply:
Whoops my bad for that suggestion :)